site_scons/extapps.scons

```diff
@@ -109,12 +110,25 @@ if appsrc := appenv.subst("$APPSRC"):
     app_artifacts = appenv.GetExtAppFromPath(appsrc)
     appenv.PhonyTarget(
         "launch_app",
-        '${PYTHON3} "${APP_RUN_SCRIPT}" "${SOURCE}" --fap_dst_dir "/ext/apps/${FAP_CATEGORY}"',
+        '${PYTHON3} "${APP_RUN_SCRIPT}" --launch "${SOURCE}" --fap_dst_dir "/ext/apps/${FAP_CATEGORY}"',
         source=app_artifacts.compact,
         FAP_CATEGORY=app_artifacts.app.fap_category,
     )
     appenv.Alias("launch_app", app_artifacts.validator)
 
+appenv.PhonyTarget(
+    "faps_copy",
+    os.linesep.join(
+        [
+            '${PYTHON3} "${APP_RUN_SCRIPT}"'
+            f' "{app_artifact.compact[0].rstr()}"'
+            f' --fap_dst_dir "/ext/apps/{app_artifact.app.fap_category}"'
+            for app_artifact in appenv["EXT_APPS"].values()
```

Comment:
That's not how you create targets in scons.
Let the build system construct the commandline and just provide a list of source nodes in `source=` parameter.

Reply:
I'm happy to get some help here as I wasn't sure what the best way is to run each copy in series. While experimenting, I did have a target with each detected application as an individual build dependency. However, scons ran them in parallel resulting in multiple script invocations attempting to connect to the Flipper via USB simultaneously. Is there a way to provide scons with a list of targets, but have them run one at a time instead of in parallel?

Reply:
I think I'll look into that myself, probably adding a multi-file mode for runfap.py.